library/src/main/java/com/cube/storm/ui/fragment/StormBottomTabsFragment.java

```diff
@@ -202,7 +201,7 @@ private void setTabItemContentDescriptions()
 			View tab = bottomNavigation.getViewAtPosition(bottomTabIdx);
 			if (tab != null)
 			{
-				String formatString = bottomNavigation.getCurrentItem() == bottomTabIdx ? getString(R.string.bottom_navigation_tab_selected) : getString(R.string.bottom_navigation_tab);
```

Comment:
What number was returning the `bottomNavigation.getCurrentItem()`, i am expecting to be the selectedTab, why is not the case?

Reply:
`bottomNavigation.getCurrentItem()` gets the previous selected tab, so I needed to use `selectedTab` variable to get there current one. If I'm on the first tab, and I would like to change to the second one, when I click, `bottomNavigation.getCurrentItem()` still gets the first tab, and `selectedTab` the second one

Reply:
So the `getCurrentItem` is not getting updated properly, ok use the fix then